componentified-todo-list/safari/src/component/TaskApp.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useEffect, useState} from "react";
+import NewTask from "./NewTask";
+import ActiveTaskList from "./ActiveTaskList";
+import DoneTaskList from "./DoneTaskList";
+
+const TaskApp: React.FC<any> = () => {
+    const [tasks, setTasks] = useState<TaskStruct[]>([])
+
+    const addNewItemTop = (subject: string) => {
+        const task: TaskStruct = {
+            id: Math.floor(Math.random() * 101),
```

Comment:
__[HIGH]__  
This is random but not unique. Use something else (like `Date.now()`).